inference-engine/thirdparty/clDNN/src/graph_optimizer/prepare_primitive_fusing.cpp

```diff
@@ -179,8 +179,10 @@ void prepare_primitive_fusing::fuse_activations(program_impl &p) {
             // - primitives input cannot be output
             // - no activation additional input
             // - input was optimized
+            // - can't have fused primitives
```

Comment:
Why? There's a branch that uses fused_ops mechanism if a node already has something fused

Reply:
I found that there is major accuracy bug when fusing activation to activation was added, which is beeing fixed by applying this change.